blog/2020-10-23-supervisor-update.md

```diff
@@ -0,0 +1,98 @@
+
+The [API documentation][api_docs] have been updated to reflect these changes.
+
+## Add-on options
+
+The permissions of the `/data/options.json` file, was change from `644` to `600`. If your add-on is running as non-root and you are reading this file, it will give now you permission issues.
+
+There are several steps you can do in your add-on to continue to use this information:
+
+- If you are using [S6-overlay] in your add-on, you can use [`/etc/fix-attrs.d`][S6-overlay-permissions] to ensure that the user you are running the add-on as, have access to the file.
+- You can change your add-on to run as `root` (default)
+
+## Releases
+
+Until now, the Supervisor, our plugins and add-ons have been using a mix of the build number and [Semantic Versioning (SemVer)][semver] as the versioning system, we have decided to replace that for these repositories and to adopt [Calendar Versioning (CalVer)][calver] as our versioning system instead.
+
+We are migrating the Supervisor from release based development to continuous development. This fits perfectly with our exists channel based update strategy (stable, beta and dev). We are now leveraging automated pipelines, to build and push out new Supervisor versions to the correct channels. By moving to this structure there was no more need to a dual branch setup, so both our `dev` and `master` branches have now been replaced with a new `main` branch. Our plugins (DNS, Multicast, Observer, CLI) for the Supervisor will also follow this continuous development principal.
+
+We did this move to provide higher software quality with automatic test system. Every commit now trigger a new dev release which get tested by our test instances and issues are imminently reported to sentry, this gives us the opportunity to test all changes before we create a releases, when a release is created the the changes are available to the beta channel. Based on the feedback and reports to Sentry we can decide to push this into stable channel.
+
+We are using [Azure pipelines][azure_pipelines] to build and publish the Supervisor. If you are interested on how we are doing this, you can look at the [release pipeline here][release_pipeline], and the [template repository here][azure_templates].
+
+## Private Container Registries
+
+In the [Supervisor 249][private-container-registries] blog post, we mentioned the introduction of private container registries. For you as a add-on maintainer this now give you the possibility to use any container registries like GitHub's package registry for docker, or your own instance of a container registry, while developing or testing your add-on.
+
+This can also be used for private testing of your add-on by giving pull access to users.
+
+To make the Supervisor use your credentials, you can use the [API][docker_api], the [CLI][cli], or the Supervisor panel in the Home Assistant UI.
+
+## GitHub Action
```

Comment:
Should this be higher in the post? Seems like the most important thing.

Reply:
It's the most exciting for sure, but most important :man_shrugging: 
Moved it to the top now.